mu4/scenes/notation/view/notationviewinputcontroller.cpp

```diff
@@ -148,11 +161,23 @@ void NotationViewInputController::startDragElements(ElementType etype, const QPo
     m_view->notationInteraction()->startDrag(els, eoffset, isDraggable);
 }
 
-void NotationViewInputController::mouseReleaseEvent(QMouseEvent* /*ev*/)
+void NotationViewInputController::mouseReleaseEvent(QMouseEvent*)
 {
     if (m_view->notationInteraction()->isDragStarted()) {
         m_view->notationInteraction()->endDrag();
+        return;
+    }
+}
+
+void NotationViewInputController::mouseDoubleClickEvent(QMouseEvent* ev)
+{
+    Element* element = m_view->notationInteraction()->selection()->element();
+
+    if (!element) {
+        return;
     }
```

Comment:
Maybe it’s better to add another check here that the element is text? For semantics, to make it clearer

Reply:
mm, yes, why not